src/main/javac/org/apache/brooklyn/tosca/a4c/brooklyn/AlienApplicationModelUpdater.java

```diff
@@ -0,0 +1,90 @@
+package org.apache.brooklyn.tosca.a4c.brooklyn;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.brooklyn.api.entity.Entity;
+import org.apache.brooklyn.api.mgmt.ManagementContext;
+import org.apache.brooklyn.core.config.ConfigKeys;
+import org.apache.brooklyn.tosca.a4c.Alien4CloudToscaPlatform;
+import org.apache.brooklyn.util.collections.MutableMap;
+
+import alien4cloud.csar.services.CsarService;
+import alien4cloud.dao.IGenericIdDAO;
+import alien4cloud.dao.MonitorESDAO;
+import alien4cloud.model.deployment.DeploymentTopology;
+import alien4cloud.model.topology.Capability;
+import alien4cloud.model.topology.NodeTemplate;
+import alien4cloud.model.topology.RelationshipTemplate;
+import alien4cloud.model.topology.Requirement;
+
+import com.google.common.collect.ImmutableMap;
+
+
+public class AlienApplicationModelUpdater {
+
+    private final ManagementContext mgmt;
+
+    public AlienApplicationModelUpdater(ManagementContext mgmt) {
+        this.mgmt = mgmt;
+    }
+
+    public void writeToAlien(Entity entity) {
```

Comment:
Does this class have any purpose over manual testing? Is it worth including? Can it be transformed into a proper test?